client/js/contextMenu.js

```diff
@@ -0,0 +1,80 @@
+"use strict";
+const $ = require("jquery");
+const templates = require("../views");
+let contextMenu, contextMenuContainer;
+
+module.exports = class ContextMenu {
+	constructor(contextMenuItems, contextMenuActions, selectedElement, event) {
+		this.contextMenuItems = contextMenuItems;
+		this.selectedElement = selectedElement;
+		this.event = event;
+
+		contextMenuContainer = $("#context-menu-container");
+		contextMenu = $("#context-menu");
+
+		contextMenuActions.execute = (id, ...args) => contextMenuActions[id] && contextMenuActions[id](...args);
+
+		contextMenu.on("click", ".context-menu-item", function() {
```

Comment:
This probably needs to change to `contextMenu.find(".context-menu-item").on("click")` and be bound after the context menu is drawn. Otherwise you have the same problem as I commented earlier.

Reply:
Alright done.